tests/resource_tests/schema_tests/test_resource_schemas.py

```diff
@@ -0,0 +1,258 @@
+#
+# This file is subject to the terms and conditions defined in the
+# file 'LICENSE', which is part of this source code package.
+#
+import json
+import unittest
+
+from tests.helpers.unittest_base import BaseTestCase
+
+# -- BQ model imports
+from rdr_service.model import (
+    bq_code, bq_genomics, bq_hpo, bq_organization, bq_participant_summary, bq_pdr_participant_summary,
+    # bq_questionnaires  <-- to do:  add tests for schemas in these files?
+    bq_site, bq_workbench_workspace, bq_workbench_researcher
+)
+
+from rdr_service.resource import schemas as rschemas
+
+# Common fields from all the BQ schemas which should be excluded from comparison.  If the resource schema contains these
+# field names, the names will be translated automatically by the pipeline to add an 'orig_' prefix for the
+# BigQuery or PostgreSQL tables
+_default_exclusions = ['id', 'created', 'modified']
+
+# Any additional "per schema" exclusions that may not be present in the corresponding resource schema
+bq_field_exclusions = {
+    'CodeSchema': ['bq_field_name'],
+    # Genomic schemas that have orig_* fields already and also id/created/modified
+    'GenomicManifestFileSchema': ['orig_id'],
+    'GenomicManifestFeedbackSchema': ['orig_id'],
+    'GenomicGCValidationMetricsSchema': ['orig_id', 'orig_created', 'orig_modified'],
+    'ParticipantSchema': ['addr_state', 'addr_zip', 'biospec']
+}
+
+# Fields from the resource schemas that do not exist in the BQ schema
```

Comment:
FYI:  I have plans to clean up how the exclusions are identified (to rely more on the pii_fields list by default for all resource schemas) but my first attempt broke the tests so I'll work on it in a future PR.